cmd/kops/upgrade_cluster.go

```diff
@@ -204,17 +205,21 @@ func (c *UpgradeClusterCmd) Run(args []string) error {
 			glog.Warningf("No matching images specified in channel; cannot prompt for upgrade")
 		} else {
 			for _, ig := range instanceGroups {
-				if ig.Spec.Image != image.Name {
-					target := ig
-					actions = append(actions, &upgradeAction{
-						Item:     "InstanceGroup/" + target.ObjectMeta.Name,
-						Property: "Image",
-						Old:      target.Spec.Image,
-						New:      image.Name,
-						apply: func() {
-							target.Spec.Image = image.Name
-						},
-					})
+				if strings.Contains(ig.Spec.Image, "kope.io") {
+					if ig.Spec.Image != image.Name {
+						target := ig
+						actions = append(actions, &upgradeAction{
+							Item:     "InstanceGroup/" + target.ObjectMeta.Name,
+							Property: "Image",
+							Old:      target.Spec.Image,
+							New:      image.Name,
+							apply: func() {
+								target.Spec.Image = image.Name
+							},
+						})
+					}
+				} else {
+					glog.Infof("Custom image (%s) has been provided for Instance Group %q; not prompting for image upgrade", ig.Spec.Image, ig.GetName())
```

Comment:
nit pick on the message, and it should be a Infof, as user is not doing something odd.

> not prompting

Is kops rather not updating the image automatically since the user is not using the kope.io image?  You have it as not prompting, which to me says CLI interaction.

Like I mentioned nit pick on english, and I am happy to get this in either way.